tests/test262/interpret-results.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+module.exports = function report(results, allowed) {
+  var expected = {
+    success: [],
+    failure: [],
+    typeI: [],
+    typeII: []
+  };
+  var unexpected = {
+    success: [],
+    failure: [],
+    typeI: [],
+    typeII: [],
+    unrecognized: null
+  };
+  var totalUnexpected;
+
+  results.forEach(function(result) {
+    var isAllowed = allowed[result.name];
+    delete allowed[result.name];
+
+    if (!!result.parseFailure === result.expected) {
```

Comment:
I feel like the logic of these if statements could be made more concise and easier to read, but it looks okay